aten/src/ATen/native/native_functions.yaml

```diff
@@ -3051,6 +3051,15 @@
   variants: method
   device_guard: False
 
+- func: vandermonde_matrix(Tensor x) -> Tensor
```

Comment:
The name in NumPy is just `vander`

Reply:
I was going to ask about this, our implementations of hamming and hann winow are called hann_window and [hamming_window](https://pytorch.org/docs/stable/torch.html#torch.hamming_window) while numpy calls them hanning and [hamming](https://docs.scipy.org/doc/numpy/reference/generated/numpy.hamming.html) 

so keep with the current pytorch pattern and use vandermonde_matrix or go with vander like numpy?

Reply:
Like NumPy. In the future we'll have NumPy names for those functions, too.